docfx_yaml/extension.py

```diff
@@ -1417,29 +1417,94 @@ def find_markdown_pages(app, outdir):
                 'href': mdfile.name.lower(),
             })
 
+def check_existing_references(
+    current_name: str,
+    words: List[str],
+    index: int,
+    word: str,
+    keyword: str,
+    converted_words: List[str],
+    hard_coded_references: Dict[str, str] = None
+):
+    """Given a word, check if it contains a reference keyword we should convert.
```

Comment:
I felt icky mixing between `None` and an empty string, however I think in this case it makes sense that we indeed return `None` instead of an empty string if a cross reference has not been found. Updating with suggestions.

Reply:
:+1: Please update this docstring. Something like `Returns the resolved cross reference for current_word, if it exists`.